services/skus/service.go

```diff
@@ -2232,6 +2254,126 @@ func (s *Service) processSubmitReceipt(ctx context.Context, req model.ReceiptReq
 	return rcpt, nil
 }
 
+const errUnknownAction = model.Error("skus: unknown radom action")
```

Comment:
suggestion: The name of the constant is too generic. Consider renaming it to `errRadomUnknownAction`. But, we are dealing with notifications/events, so the term `action` seems unnecessary here. And also, all other similar methods don't fail when received an event which should not be processed. So I argue that this error is not needed at all.

Reply:
Events trigger behaviours i.e actions that can be performed, in this case a new, renewal or cancellation of a subscription. This error signifies that the event is attempting to triggering an unknown action. 

Taken in the context of `processRadomEvent` and a successful call to [should process](https://github.com/brave-intl/bat-go/pull/2640/files#diff-225dad392327a49c7aa0c5287edf78e66ae94759a42c08d4207299c0e0d6e895R2260) then you would expect it to execute a behaviour otherwise throw an error. Returning an error from `processRadomEventTx` and using the a separate method i.e. allows the caller to decide and makes testing easier rather than all in `processRadomEvent`.

Agreed, the name could be less generic e.g. `errRadomUnknownAction`

Reply:
Yes, agreed. I will have to update the other similar methods to be in line with this as well.